Add line about dismiss feature in recs _index.md (#32270)
DataDog/documentation: content/en/cloud_cost_management/recommendations/_index.md

- **Jira**: Jira issue creation is available in both the recommendation side panel and the "Active Recommendations" list. You can create a Jira issue by clicking "Create Jira issue" in the side panel or by selecting multiple recommendations in the "Active Recommendations" list. Created Jira issues are automatically tagged to indicate their connection to a cost recommendation and include a link back to the referenced recommendation.
- **1-click Workflow Automation actions**: Actions are available for a limited set of recommendations, allowing users to execute suggested actions, such as clicking "Delete EBS Volume", directly within Cloud Cost Management.
- **Datadog Case Management**: Users can go to the recommendation side panel and click "Create Case" to generate a case to manage and take action on recommendations.

> - **Dismiss**: Users can go to the recommendation side panel and click "Dismiss" to dismiss a recommendation for a selected time frame and give a reason for the dismissal. Dismissed recommendations appear in the "Dismissed" tab.

Reviewer: feel free to ignore, but this suggestion reduces the repeated "dismiss" verb a bit!

> - **Dismiss**: Users can go to the recommendation side panel and click "Dismiss" to dismiss a recommendation for a selected timeframe and give a reason for the dismissal. Dismissed recommendations will then appear in the "Dismissed" tab.

Reviewer 2: ```suggestion
- **Dismiss**: Users can go to the recommendation side panel and click "Dismiss" to dismiss a recommendation for a selected time frame and give a reason for the dismissal. Dismissed recommendations appear in the "Dismissed" tab.
```
